aula-2/src/main/java/org/acelera/brasil/futebol/JogadorBuilder.java

```diff
@@ -0,0 +1,51 @@
+package org.acelera.brasil.futebol;
+
+/**
+ * Classe responsável pela construção de um {@link Jogador}
+ * 
+ * @author edmilson.santana
+ *
+ */
+public class JogadorBuilder {
+
+	private String nome;
+
+	private String cidade;
+
+	private String pais;
+
+	private Posicao posicao;
+
+	private Integer numeroDeGols;
+
+	protected JogadorBuilder() {}
+	
+	public JogadorBuilder withNome(String nome) {
+		this.nome = nome;
+		return this;
+	}
+
+	public JogadorBuilder withCidade(String cidade) {
+		this.cidade = cidade;
+		return this;
+	}
+
+	public JogadorBuilder withPais(String pais) {
+		this.pais = pais;
+		return this;
+	}
+
+	public JogadorBuilder withPosicao(Posicao posicao) {
+		this.posicao = posicao;
+		return this;
+	}
+
+	public JogadorBuilder withNumeroDeGols(Integer numeroDeGols) {
```

Comment:
:question: O que acontece se o número de gols for negativo?

Reply:
Hmmm.. não testei nem tratei esse caso, vou verificar!